docker-compose.yaml

```diff
@@ -179,6 +181,65 @@ services:
     volumes:
       - ./cdr/cdm/jupyter/user_shared_workspace:/cdm_shared_workspace/user_shared_workspace
 
+  dev_jupyterhub:
+      build:
+        context: .
+        dockerfile: Dockerfile
+      container_name: dev-jupyterhub
+      ports:
+        - "4043:4043"
+      depends_on:
+        - spark-master
+        - minio-create-bucket
+      environment:
+        - NOTEBOOK_PORT=4043
+        - JUPYTER_MODE=jupyterhub
+        - YARN_RESOURCE_MANAGER_URL=http://yarn-resourcemanager:8032
+        - SPARK_MASTER_URL=spark://spark-master:7077
+        - SPARK_DRIVER_HOST=dev-jupterhub
+        - MINIO_URL=http://minio:9002
+        - MINIO_ACCESS_KEY=minio-readwrite
+        - MINIO_SECRET_KEY=minio123
+        - S3_YARN_BUCKET=yarn
+        - MAX_EXECUTORS=4
+        - POSTGRES_USER=hive
+        - POSTGRES_PASSWORD=hivepassword
+        - POSTGRES_DB=hive
+        - POSTGRES_URL=postgres:5432
+        - USAGE_MODE=dev
+      volumes:
+        - ./cdr/cdm/jupyter:/cdm_shared_workspace
+        - ./cdr/cdm/jupyter/home:/home
```

Comment:
I'd make this specific to jhub to avoid confusion

Reply:
👍

Reply:
:+1: